README.md

```diff
@@ -68,6 +68,11 @@ __Important__ - _all_ AEP SDK Android binaries depend on `core-1.x.x.aar`.
 
 ## Installing the AEP SDK for iOS
 
+> __Important__ 
+> 
+> ⚠️ A newer version of the [AEP Mobile SDKs (3.x)](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift) is now available! These SDKs are open source, bring many exciting new features and include support for Swift Package Manager (SPM), CocoaPods and XCFrameworks.
```

Comment:
As this message is already under the "Important" heading, I don't think the emoji is needed, unless it is added to all the "Important" messages.

Reply:
Made few updates so that this note stands out, let me know what you think.